mkdocs/docs/SUMMARY.md

```diff
@@ -26,6 +26,6 @@
 - Releases
     - [Verify a release](verify-release.md)
     - [How to release](how-to-release.md)
-- [Code Reference](reference/)
+- [Code Reference](reference/) <!-- markdown-link-check-disable-line -->
```

Comment:
I tried to build the site locally but received the following error
```
mkdocs_literate_nav.parser.LiterateNavParseError: Expected no text after <a href="reference/">Code Reference</a>, but got ' <!-- markdown-link-check-disable-line -->'.
```
Seems we also need to use `<!-- markdown-link-check-disable -->`/`<!-- markdown-link-check-enable-->` here?

Reply:
good catch! I was able to verify that the updated PR can build the site successfully

```
cd mkdocs
mkdocs build --strict                                               
```